lib/freedom_patches/request_secure_session.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module RequestSecureSession
+  def secure_session
+    SecureSession.new(session[:secure_session_id] ||= SecureRandom.hex)
+  end
+end
+
+Rack::Request::Helpers.prepend(RequestSecureSession)
```

Comment:
This makes `secure_session` available everywhere we have a `request` object (either coming from Rack or ActionDispatch).